From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: sidex15 <24408329+sidex15@users.noreply.github.com>
Date: Fri, 11 Jul 2025 09:29:50 +0800
Subject: [PATCH 2/5] [PATCH 2/5] susfs: drop
 susfs_umount_for_zygote_iso_service

Remove file /data/adb/susfs_umount_for_zygote_iso_service in case of any infinite bootloop without being able to boot into ksu rescue mode, see post-fs-data.sh in module template for proer usage
---
 include/linux/susfs_def.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/linux/susfs_def.h b/include/linux/susfs_def.h
index de67737e0293..09693d08f0dc 100644
--- a/include/linux/susfs_def.h
+++ b/include/linux/susfs_def.h
@@ -67,7 +67,6 @@
 
 #define MAGIC_MOUNT_WORKDIR "/debug_ramdisk/workdir"
 #define DATA_ADB_UMOUNT_FOR_ZYGOTE_SYSTEM_PROCESS "/data/adb/susfs_umount_for_zygote_system_process"
-#define DATA_ADB_UMOUNT_FOR_ZYGOTE_ISO_SERVICE "/data/adb/susfs_umount_for_zygote_iso_service"
 #define DATA_ADB_NO_AUTO_ADD_SUS_BIND_MOUNT "/data/adb/susfs_no_auto_add_sus_bind_mount"
 #define DATA_ADB_NO_AUTO_ADD_SUS_KSU_DEFAULT_MOUNT "/data/adb/susfs_no_auto_add_sus_ksu_default_mount"
 #define DATA_ADB_NO_AUTO_ADD_TRY_UMOUNT_FOR_BIND_MOUNT "/data/adb/susfs_no_auto_add_try_umount_for_bind_mount"
